Makefile

```
@@ -338,6 +338,20 @@ test.integration.kind:
 	@$(MAKE) _test.integration.cp \
 		CP="kind"
 
+.PHONY: test.e2e.gke
+test.e2e.gke:
+	CLUSTER_NAME="e2e-$(uuidgen)" \
+		KUBERNETES_CLUSTER_NAME="${CLUSTER_NAME}" go run hack/e2e/cluster/deploy/main.go \
+		KONG_TEST_CLUSTER="gke:${CLUSTER_NAME}" \
+		GOFLAGS="-tags=e2e_tests" go test -v $(GOTESTFLAGS) \
```

```suggestion
		GOFLAGS="-tags=e2e_tests" $(GOTESTSUM) -- $(GOTESTFLAGS) \
```

Run with gotestsum in order to generate the JUnit report.